src/Microsoft.IdentityModel.Tokens/TokenContext.cs

```diff
@@ -0,0 +1,54 @@
+﻿//------------------------------------------------------------------------------
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Microsoft.IdentityModel.Tokens
+{
+    /// <summary>
+    /// An opaque context used to store work when working with authentication artifacts.
+    /// </summary>
+    [Obsolete("The 'TokenContext' property is obsolete. Please use 'CallContext' instead.")]
+    public class TokenContext : CallContext
+    {
+        /// <summary>
+        /// Instantiates a new <see cref="TokenContext"/> with a default activityId.
```

Comment:
Nit: activityId -> activity ID. (Same below.)

Reply:
[Fixed](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1331/commits/1db9d409a7866b1785344ec54802cf5c1e0ccef7#diff-cc3db3b84c666213be06d65df1f12b82)!